### ISSUE TITLE
State and country boundaries not showing

### ISSUE THREAD

**Issue by brewin** [mapzen/vector-datasource#517]
I'm not able to get any admin boundaries to show up using Mapbox GL. The [example](https://mapzen.github.io/mapboxgl-vector-tiles/) isn't showing them either even though they're in the style.

```json
{
      "id": "country-boundary",
      "source": "osm",
      "source-layer": "places",
      "type": "line",
      "filter": ["==", "admin_level", "2"],
      "max-zoom": 4,
      "layout": {
        "line-cap": "round",
        "line-join": "round"
      },
      "paint": {
        "line-color": "@building",
      "line-width": {
        "base": 2,
        "stops": [[1, 0.5], [7, 3]]
        }
      }
    }, {
      "id": "state-boundary",
      "source": "osm",
      "source-layer": "places",
      "type": "fill",
      "filter": ["==", "admin_level", "4"],
      "max-zoom": 10,
      "layout": {
        "line-cap": "round",
        "line-join": "round"
      },
      "paint": {
        "fill-color": "@land",
        "fill-outline-color": "#cacecc"
      }
    }
```

**Comment by zerebubuth** [comment]
You're right! That looks like the wrong layer. I think the `source-layer` should be `boundaries` instead of `places`. Could you try that and let us know if it works?

**Comment by brewin** [comment]
Still nothing with `boundaries`.

**Comment by brewin** [comment]
I figured out how to get them to show, but I don't think it's correct.

```json
"source-layer": "boundaries",
"filter": ["==", "type", "country"],
```

I think `type` should be `kind` for consistency, and filtering by `admin_level` doesn't work. Also, there is no mention of the `boundaries` layer in the [docs](https://mapzen.com/documentation/vector-tiles/layers/).

**Comment by nvkelso** [comment]
Thanks for digging into this! I'll have a closer look on Monday. 

Background: There are a few inconsistencies like this today, and normalizing them is something we're committed to for a 1.0 release. It's super helpful to hear feedback like this.

**Comment by nvkelso** [comment]
Related, our MapboxGL demo should be updated:

https://github.com/mapzen/mapboxgl-vector-tiles/issues/16

**Comment by nvkelso** [comment]
From the docs:

**Gotchas:**

* `Admin-0 country` and `country` are equivelant, both should be used in filters.
* `Admin-1 boundary` and `state` are equivelant, both should be used in filters.
* Don’t rely on `Admin-1 statistical boundary`, `Admin-1 statistical meta bounds`, those will probably be removed in future tile versions.
